mappings/net/minecraft/client/gl/VertexBuffer.mapping

```diff
@@ -6,22 +6,35 @@ CLASS net/minecraft/class_291 net/minecraft/client/gl/VertexBuffer
 	FIELD field_27368 drawMode Lnet/minecraft/class_293$class_5596;
 	FIELD field_29338 vertexArrayId I
 	FIELD field_29339 vertexFormat Lnet/minecraft/class_293;
+	FIELD field_38983 indexBuffer Lcom/mojang/blaze3d/systems/RenderSystem$class_5590;
 	METHOD method_1352 upload (Lnet/minecraft/class_287;)V
 		ARG 1 buffer
 	METHOD method_1353 bind ()V
 	METHOD method_1354 unbind ()V
 	METHOD method_22643 submitUpload (Lnet/minecraft/class_287;)Ljava/util/concurrent/CompletableFuture;
 		ARG 1 buffer
 	METHOD method_22644 uploadInternal (Lnet/minecraft/class_287;)V
-	METHOD method_34427 setShader (Lnet/minecraft/class_1159;Lnet/minecraft/class_1159;Lnet/minecraft/class_5944;)V
+		ARG 1 builder
+	METHOD method_34427 draw (Lnet/minecraft/class_1159;Lnet/minecraft/class_1159;Lnet/minecraft/class_5944;)V
```

Comment:
I think we need to distinguish from the existing `drawElements` method. The difference is whether it uses a shader.